api/src/main/java/org/commonjava/indy/cluster/IndyNode.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (C) 2013~2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.cluster;
+
+import java.io.Serializable;
+
+/**
+ * A data model to describe a clustered indy node.
+ */
+public class IndyNode
```

Comment:
Is there a reason this is preferable to a String?

Reply:
I'm not sure if this node info will contain some other stuff in future (like ip or anything else?), so I used a model here for future extension. But for now, it is good enough for a simple string.